### ISSUE TITLE
Accessibility: Whistleblower report review page issues

### ISSUE THREAD

**Issue by nicolaisweb** [globaleaks/GlobaLeaks#3573]
### What version of GlobaLeaks are you using?

4.12.8

### What operating system(s) are you seeing the problem on?

Windows, macOS, Android, iOS, Linux

### Describe the issue

1. In the whistleblower report review page there are `<div>` tags with the same id `#TipInfoBox`.
2. In the whistleblower report review page the "Refresh" button doesn't have a description (add aria-label).
3. In the whistleblower report review page the "Comments" `<textarea>` doesn't have a `<label>` tag.

### Proposed solution

1. Assign unique IDs to the elements in the page
2. Add aria-label attribute to the refresh button element
3. Add a label referring to the textarea ID
